Initium-Odp/src/com/universeprojects/miniup/server/ODPDBAccess.java

```diff
@@ -1627,6 +1627,18 @@ public boolean awardBuff_Berry(CachedDatastoreService ds, CachedEntity character
 		}
 		return buffApplied;
 	}
+	
+	public boolean awardBuff_Elixir(CachedDatastoreService ds, CachedEntity character)
+	{
+		boolean buffApplied = false;
+		CachedEntity buff = awardBuff(ds, character.getKey(), "http://i.imgur.com/vCbVFpw.png","Strange Elixir",
```

Comment:
He has it figured out now though right? There isn't much reason to have a placeholder like this anymore with the Images repo really.. Even if it's not live now, it will be when I deploy this change anyway.